eiffel-vocabulary/EiffelConfigurationOrderedEvent.md

```diff
@@ -0,0 +1,19 @@
+
+# EiffelConfigurationOrderedEvent (ArtS)
+
+The EiffeliConfigurationOrderedEvent declares that a configuration has been Ordered.
+
+## Data Members
+### data.reason
+__Type:__ String  
+__Required:__ No  
+__Description:__ The EiffeliConfigurationOrderedEvent declares that a configuration has been Ordered.
```

Comment:
same as above .,...  the data.reason is not accurate and simple filler to get me started.